app/services/providers/visa_sponsorship_service.rb

```diff
@@ -0,0 +1,14 @@
+module Providers
+  class VisaSponsorshipService
+    VISA_SPONSORSHIP_INTRODUCED_IN = 2022
+
+    def initialize(provider)
+      @provider = provider
+    end
+
+    def visa_sponsorship_enabled?
+      @provider.recruitment_cycle_year.to_i >= VISA_SPONSORSHIP_INTRODUCED_IN &&
+        Settings.features.rollover.prepare_for_next_cycle
```

Comment:
Yea i do agree that using

```
provider.recruitment_cycle.year.to_i < VISA_SPONSORSHIP_REQUIRED_FROM
```

seems enough to me.

I don't really see the advantage of stuff being behind a flag because it should be visible for all rolled over course in the 2022 cycle and it's not like we're going to be switching it off. WDYT Steve?

Reply:
Happy to drop the flag.